src/elm/Profile/Summary.elm

```diff
@@ -150,7 +163,7 @@ viewUserImg shared loggedInAccount profile isMobile model =
             [ container [] [ Avatar.view profile.avatar model.imageSize ]
             , if not isMobile && model.isExpanded then
                 View.Components.dialogBubble
-                    { class_ = "w-120"
+                    { class_ = "w-120 animate-fadeIn opacity-0"
```

Comment:
After changing the tailwind config, we need to change it here too, to use `animate-fade-in` (right now, the popup doesn't even show)
```suggestion
                    { class_ = "w-120 animate-fade-in opacity-0"
```